source/Grid/Grid.js

```diff
@@ -264,8 +264,11 @@ export default class Grid extends PureComponent {
     this._invokeOnGridRenderedHelper = this._invokeOnGridRenderedHelper.bind(this)
     this._onScroll = this._onScroll.bind(this)
     this._setScrollingContainerRef = this._setScrollingContainerRef.bind(this)
+    this._getCalculatedScrollLeft = this._getCalculatedScrollLeft.bind(this)
     this._updateScrollLeftForScrollToColumn = this._updateScrollLeftForScrollToColumn.bind(this)
+    this._getCalculatedScrollTop = this._getCalculatedScrollTop.bind(this)
     this._updateScrollTopForScrollToRow = this._updateScrollTopForScrollToRow.bind(this)
+    this._setScrollPosition = this._setScrollPosition.bind(this)
```

Comment:
I don't see why these 3 new functions are bound. They're internal; they're only called from other methods that have the correct context (`this`). It doesn't seem like this binding is necessary.

The only reason methods like `_onScroll` and `_setScrollingContainerRef` are bound this way is that they're passed as properties to React and may not be called with the correct context.

While writing this comment I noticed that `_updateScrollLeftForScrollToColumn` and `_updateScrollTopForScrollToRow` no longer need to be bound since they aren't passed directly to `updateScrollIndexHelper` anymore. 😄